src/test/suite/extension.test.ts

```diff
@@ -9,4 +9,13 @@ suite('Extension Test Suite', () => {
 		done();
 	});
 
+	test.skip('look for key binding conflict with Ctrl+Shift+V', async function() {
+		try {
+			await vscode.commands.executeCommand('keybindings.editor.showConflicts', "Control+Shift+V").then( (result) => {
```

Comment:
based on https://code.visualstudio.com/docs/getstarted/keybindings#_accepted-keys I would expect that it would be written Ctrl+Shift+V

Reply:
When I looked at what was returned by the visual search for conflicts, it was "Control+Shift+V" so that's how I did it